### PR TITLE
Convert to an ember-addon

### PR DESCRIPTION
**Pull request by trentmwillis**
This works to get all the files available properly, but has one caveat. If we try to use it as a dependency of `ember-cli-qunit`, it won't work properly. The reason being the `included` hook for this addon will run before `ember-cli-qunit` which results in QUnit being concatenated into `vendor.js` after `qunit-notifications` which breaks.

One alternative is to decouple this from `ember-cli-qunit` and just require that QUnit make it into vendor somehow.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ember-qunit-notifications",
+  "version": "1.0.0",
```
Comment (bcardarella) [inline package.json]: Can you change this to `0.1.0` to match the current release?

**Comment by bcardarella** [comment]
I actually always thought this would end up getting pulled into `ember-cli-qunit`. Perhaps that is the best approach?

**Comment by trentmwillis** [comment]
> I actually always thought this would end up getting pulled into ember-cli-qunit

As in merge the code into `ember-cli-qunit`?

I would actually favor decoupling the two as this adds non-default APIs to QUnit that aren't necessary to testing Ember applications.

@stefanpenner @rwjblue would appreciate your input here.

**Comment by bcardarella** [comment]
This library only serves to hold the ember-specific pass/fail images so it wouldn't add any weight to `ember-cli-qunit` outside of adding `qunit-notifications` as a dependency. It would seem best to package all of the qunit-specific addons together as a single choke-point rather than having several.

**Comment by stefanpenner** [comment]
combining them seems reasonable from my perspective, although i could be convinced either way

**Comment by trentmwillis** [comment]
> It would seem best to package all of the qunit-specific addons together as a single choke-point rather than having several.

I think `ember-cli-qunit` should be the single point for facilitating all _necessary_ code to enable QUnit testing in an Ember application. Lumping this addon in with the core functionality defeats the purpose of having separate addons (at least in my mind); consumers should be allowed to choose which extended functionality they are absorbing into their application. While this addon itself is not substantial, what is our criteria for saying other QUnit extensions don't also get bundled into `ember-cli-qunit` by default?

If we feel that this functionality is core to the Ember QUnit testing story, then we can merge it, but I'm not convinced that it is (as, anecdotally, I've never seen anyone use it).

**Comment by stefanpenner** [comment]
@trentmwillis I think if we bundle it by default, and it is used by many (and removed by very few) we should consider bundling. Purely to simplify. 

Ultimately, I prefer whatever steps get this off bower the quickest ;)

**Comment by bcardarella** [comment]
if @rwjblue is on board I can PR `ember-cli-qunit`

**Comment by trentmwillis** [comment]
> Ultimately, I prefer whatever steps get this off bower the quickest

Okay, bundling is the best option in that case, since decoupling them would probably be a backwards-incompatible change and be more of a headache to roll out. I'll look into doing it later today.

**Pull request closed without merging**